Source/ASDisplayNode.mm

```diff
@@ -2635,6 +2635,36 @@ - (BOOL)placeholderShouldPersist
   return NO;
 }
 
+- (BOOL)placeholderEnabled
+{
+    return _placeholderEnabled;
+}
+
+- (void)setPlaceholderEnabled:(BOOL)placeholderEnabled
+{
+    _placeholderEnabled = placeholderEnabled;
+}
+
+- (NSTimeInterval)placeholderFadeDuration
+{
+    return _placeholderFadeDuration;
+}
+
+- (void)setPlaceholderFadeDuration:(NSTimeInterval)placeholderFadeDuration
+{
+    _placeholderFadeDuration = placeholderFadeDuration;
+}
+
+- (NSInteger)drawingPriority
+{
+    return _drawingPriority;
+}
+
+- (void)setDrawingPriority:(NSInteger)drawingPriority
+{
+    _drawingPriority = drawingPriority;
```

Comment:
Discussed with @garrettmoon @nguyenhuy and these properties do need locks. I'll also see if I can always enable this warning as error in this class. This way we can get these warnings enabled, and fixed 1 by 1 until they are able to be be enabled for the entire project.